lib/ExecutionEngine/ExecutionEngine.cpp

```diff
@@ -79,6 +79,21 @@ void glow::updateVariables(Context &ctx, llvm::ArrayRef<Placeholder *> ph,
   }
 }
 
+void glow::updateVariablesbyName(Context &ctx, Module *mod,
+                                 llvm::ArrayRef<llvm::StringRef> ph,
```

Comment:
Maybe the function name can be changed to something like "updatePlaceholderByName" ? Now the "Variables" always make me think of "Constant" :)

Reply:
Good call. I renamed the method to updateInputsByName.